### DISCUSSION TITLE
有一说一,LyricEase的UI确实很好看

### DISCUSSION THREAD

**Discussion by kengwang** [HyPlayer/HyPlayer#38] [General]
页面确实很好看,不过不能登录
还有就是歌词的虚化我不太喜欢
![image](https://user-images.githubusercontent.com/30862240/120968056-f1c45900-c79a-11eb-8797-f314dcd918df.png)
![image](https://user-images.githubusercontent.com/30862240/120968130-0dc7fa80-c79b-11eb-9dde-c088590adf80.png)

**Comment by ApigeonAPE** [comment]
HY的UI快赶上LE了（个人感受），就是部分还有问题，比如播放条作为单独的组件浮在最上层， _居然不能和底部完美贴合？_ 留下的一条好难受。
![image](https://user-images.githubusercontent.com/61077394/154501230-bc516955-7041-4950-a9d9-c97466142391.png)
还有就是按钮有点丑（原谅我太挑剔）

**Reply by kengwang** [reply]
要不这个作为可选选项 /笑
啊不, 我是认真的, 很多人反馈这个地方的设计, 不过确实从一开始就是这样打算的.

新版本 (2.1.x) 将会把 PlayBar 的颜色和后面区分开 并且加入适当的阴影.
并且会给一个设置选项 是否 PlayBar 周围留白

**Reply by ApigeonAPE** [reply]
> 
想到一块去了！
本来我打算写个discussion就提颜色区分的建议的，结果~~垃圾~~Github抽风下线，所以就放弃了。

**Reply by kengwang** [reply]
这个在 729623e355e0d0db09523fce8ffaa6cfc37d121e 中添加了设置项到实验室中 `PlayBar 悬浮`, 可将其取消勾选

**Reply by kengwang** [reply]
![image](https://user-images.githubusercontent.com/30862240/155869226-aa2fa23b-945f-4d86-92bc-f5271c89ca93.png)

**Reply by ApigeonAPE** [reply]
赞👍

**Comment by XiaoTDM** [comment]
LyricEase 彻底登不上去了，类 Apple Music 的界面我是真的喜欢，网易真的不干人事
